src/test/java/com/spotify/docker/client/DefaultDockerClientTest.java

```diff
@@ -1896,15 +2060,10 @@ private EventStream getImageAndContainerEventStream(final EventsParam... eventsP
   }
 
   @SuppressWarnings("deprecation")
-  private void eventStreamAssertions(final Event pullEvent,
-                                     final Event createEvent,
-                                     final Event startEvent,
-                                     final String containerId,
-                                     final String imageName) throws Exception {
-    // pull event assertions
+  private void imagePullEventAssertions(final Event pullEvent) throws Exception {
     assertThat(pullEvent.time(), notNullValue());
     if (dockerApiVersionAtLeast("1.22")) {
-      assertEquals("image", pullEvent.type());
+      assertEquals(IMAGE.getName(), pullEvent.type());
```

Comment:
I left the `IMAGE.getName()` hard-coded. The method is a helper for assertions about image events only, so they will always have that `event.type()`. But I generalized the image name and the type of action.